debug/set_up_keyvault.sh

```diff
@@ -0,0 +1,136 @@
+export CERT2_VALUE=$(cat "$CERT_PATH/${CERT2_NAME}_decoded".pem | base64 -w 0)
+export CERT2_SECRET_VALUE=$(cat "$CERT_PATH/${CERT2_NAME}.pem" | base64 -w 0)
+export CERT2_KEY_VALUE=$(openssl x509 -pubkey -noout -in "$CERT_PATH/${CERT2_NAME}.pem"  | base64 -w 0)
+
+
+echo "\n Setting up ECC Cert \n"
+export CERT3_NAME=eccCert
+openssl ecparam -name prime256v1 -out "$CERT_PATH/${CERT3_NAME}.prime256v1.param.pem"
+openssl req -new -x509 -sha256 -newkey ec:"$CERT_PATH/${CERT3_NAME}.prime256v1.param.pem" -nodes -keyout "$CERT_PATH/${CERT3_NAME}.prime256v1.key.pem" -days 365 -out "$CERT_PATH/${CERT3_NAME}.prime256v1.cert.pem"
+openssl pkcs12 -export -keysig -out "$CERT_PATH/${CERT3_NAME}.prime256v1.cert.pfx" -inkey "$CERT_PATH/${CERT3_NAME}.prime256v1.key.pem" -in "$CERT_PATH/${CERT3_NAME}.prime256v1.cert.pem"
+az keyvault certificate import --file "$CERT_PATH/${CERT3_NAME}.prime256v1.cert.pfx" --name $CERT3_NAME --vault-name $KEYVAULT_NAME
+# Add ECCCert to secrets.env
+cat "$CERT_PATH/${CERT3_NAME}.prime256v1.key.pem" "$CERT_PATH/${CERT3_NAME}.prime256v1.cert.pem" > "$CERT_PATH/${CERT3_NAME}.pem"
+export CERT3_VALUE=$(cat "$CERT_PATH/${CERT3_NAME}.prime256v1.cert.pem" | base64 -w 0)
+export CERT3_SECRET_VALUE=$(cat "$CERT_PATH/${CERT3_NAME}.pem" | base64 -w 0)
+export CERT3_KEY_VALUE=$(openssl x509 -pubkey -noout -in "$CERT_PATH/${CERT3_NAME}.pem" | base64 -w 0)
+
+echo "\n Setting Object 1 in Key Vault \n"
+export OBJECT1_NAME=secret1
+export OBJECT1_VALUE=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)
+export OBJECT1_TYPE=secret
+export OBJECT1_ALIAS=SECRET_1
+export OBJECT1_VERSION=
+az keyvault secret set --vault-name $KEYVAULT_NAME --name $OBJECT1_NAME --value $OBJECT1_VALUE
+
+echo "\n Setting Object 2 in Key Vault \n"
+export OBJECT2_NAME=secret2
+export OBJECT2_VALUE=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)
+export OBJECT2_TYPE=secret
+export OBJECT2_ALIAS=SECRET_2
+export OBJECT2_VERSION=
+# geberate pem cert
+az keyvault secret set --vault-name $KEYVAULT_NAME --name $OBJECT2_NAME --value $OBJECT2_VALUE
+# az keyvault key import --vault-name $KEYVAULT_NAME --name $OBJECT2_NAME --pem-file "$CERT_PATH/${CERT1_NAME}.pem"
+# Key val should be same as CERT1_KEY_VALUE
+
+echo "\n Creating Service Principal and Assigning Roles \n"
+# Create Service Principal
+export AZURE_CLIENT_ID=$(az ad sp create-for-rbac --name "$RG_NAME-sp" --role="Reader" --scopes $KEYVAULT_ID --query "{appId: appId}" -o tsv)
+export AZURE_CLIENT_SECRET=$(az ad sp credential reset --name $RG_NAME-sp --credential-description $RG_NAME --query password -o tsv)
+# Add credentials to K8s cluster
+# kubectl create secret generic secrets-store-creds --from-literal clientid=$AZURE_CLIENT_ID --from-literal clientsecret=$AZURE_CLIENT_SECRET
+
+# assign roles and policies
+az role assignment create --role Reader --assignee $AZURE_CLIENT_ID --scope $KEYVAULT_ID
```

Comment:
The SP is created with `Reader` role on `KEYVAULT_ID`. I think we can remove this role assignment, right?